mitosheet/mitosheet/step_performers/set_cell_value.py

```diff
@@ -27,6 +27,9 @@ class SetCellValueStepPerformer(StepPerformer):
     """
     A set_cell_value step, allows you to set the value
     of a given cell in the sheet.
+
+    NOTE: this step performer is no longer generated by the frontend, so we don't worry
```

Comment:
Makes sense. Maybe at some point we should upgrade `set cell value` steps to the steps? Not a high priority though

Reply:
Perhaps. It's also fine to just leave it around for a year, and eventually just kill it (probably).

Reply:
Probably. Might require some fancy type working. But yea.